README.md

```diff
@@ -0,0 +1,24 @@
+# Civ
+
+This monorepo will eventually contain all civ projects and development
+
+## Developing Locally
+
+### Plugins
+
+### Containers
+A docker compose stack is provided to help test containers built from
+this repo. To start the stack, run the following command:
+`docker compose up --build`. Please note that this stack is NOT suitable for production use.
+
+Optional services may be started by enabling the profile flag, e.g. `--profile <name>`
+
+Current services and exposed ports are:
+
+| Name     | Ports     | Profile    |
+|----------|-----------|------------|
+| pvp      |           |            |
+| mariadb  | TCP/3306  |            |
+| postgres | TCP/5432  |            |
+| rabbitmq | HTTP/5672 |            |
+| grafana  | HTTP/3000 | monitoring |
```

Comment:
should be all tcp

Reply:
I wanted to specifically call out the service not the protocol to clue people into that being what it is, especially since people might not be familiar with the rabbitmq admin gui, but I can change this to be more clear

Reply:
Might be worth adding the RabbitMQ dashboard port too: [15672](https://www.rabbitmq.com/docs/management#usage-ui)